### PR TITLE
PAY-850: Revise Payouts doc to reflect date picker changes

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) revised the Payouts topic to reflect the new calendar/date picker changes and process for choosing payout transactions dates.

Branch for merging into integration branch.

Jenkins preview 39/user-guide/magento-payments/payouts.html

https://jira.corp.magento.com/browse/PAY-850

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- ...
- ...

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

File: src/magento-payments/payouts.md [hunk 1]
```diff
@@ -13,42 +13,36 @@ You can download payout transactions in a .csv file format for use in existing a
...
 
-If you want to see all payout transactions for a time period other than the default last 30 days, you can customize the payout transaction timeframe for specified dates.
+From the Payouts view, you can customize the timeframe for the payout transactions you want to view by entering a specific timeframe or selecting a date range from the date picker:
 
 1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts**.
-1. Click the Payout transaction dates calendar selector filter.
-1. Select the desired timeframe for your report---_Yesterday_, _Last 7 days_, _Last 30 days_, _Last 90 days_, _Year to date_, or _Custom range_. See the [Default transactions timeframes section](#default-transactions-timeframes) for more information.
-   - If you selected _Custom range_, choose the applicable date range.
-1. View the payout transactions for your specified dates in the grid.
+1. Enter in a date range, such as `2021-01-02 - 2012-01-12`, in a YEAR-MONTH-DAY format, or select a specific date range from the date picker.
```
Comment (rancollier) [inline src/magento-payments/payouts.md:32]: I am not sure if YYYY-MM-DD is a universal format that most people would understand in place of YEAR-MONTH-DAY

Comment (shrielenee) [reply src/magento-payments/payouts.md:32]: Good point. I'll check into that and modify.

File: src/magento-payments/payouts.md [hunk 2]
```diff
@@ -13,42 +13,36 @@ You can download payout transactions in a .csv file format for use in existing a
...
 By default, 30 days of transactions are shown in the grid.
 
 Scroll to the left and right to view [information for each payout transaction](#column-descriptions) in the daily report, including transaction date, reference ID, invoice number, and payment method details.
 
-The number of payout transactions that are visible, for the default 30 day timeframe or a [customized timeframe](#customize-transactions-timeframe), are shown above the payouts grid as a records count.
-
 ### Customize transactions timeframe
 
-If you want to see all payout transactions for a time period other than the default last 30 days, you can customize the payout transaction timeframe for specified dates.
+From the Payouts view, you can customize the timeframe for the payout transactions you want to view by entering a specific timeframe or selecting a date range from the date picker:
```
Comment (rancollier) [inline src/magento-payments/payouts.md:29]: I am not fond of 'timeframe'. 
We can either enter a date range through the input fields or select a date range from the calendar date picker.
Maybe:
From the Payouts view, you can customize the date range for payout transactions you want to view by entering specific dates or selecting a range from the date picker.

**Review by rancollier — APPROVED** [review]

**Comment by shrielenee** [comment]
Since this is a revision of an existing topic that was already approved by Product, I will merge without their approval.

**Pull request merged**